swift/llm/utils/utils.py

```diff
@@ -332,6 +354,16 @@ def data_collate_fn(batch: List[Dict[str, Any]],
         attention_mask, batch_first=True, padding_value=0)
     labels = pad_sequence(labels, batch_first=True, padding_value=-100)
 
+    if padding_to is None and use_torchacc():
```

Comment:
Can I ask if padding_to and bucket_sizes cannot be set simultaneously

Reply:
Yes, bucketing of sequence lengths is only needed when sequence length is dynamic i.e. padding_to is None.